SFBaseKitDemo/SFBaseKitDemo/ExampleHomeModule/HomeViewController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  HomeViewController.swift
+//  SFBaseKitDemo
+//
+//  Created by Aleksandar Geyman on 21.04.20.
+//  Copyright © 2020 Upnetix. All rights reserved.
+//
+
+import SFBaseKit
+
+class HomeViewController: UIViewController {
+    
+    unowned var sceneDelegate: HomeSceneDelegate!
```

Comment:
Move HomSceneDelegate protocol to be defined in this class. (this is the rule we discussed on the weekly meeting)

Reply:
I've managed to watch the recording. Moved the protocol in the view controller file.